docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -106,6 +106,30 @@ export S3_ENDPOINT_URL='https://<accountid>.r2.cloudflarestorage.com'
 ```
 ````
 
+One can also set credentials through environment variables. For example, with Backblaze:
+
+```{note}
+Your endpoint url is 'https://s3.<your region>.backblazeb2.com'. The account ID can be retrieved through your [Backblaze console](https://secure.backblaze.com/b2_buckets.htm).
+```
+
+````{tabs}
+```{code-tab} py
+import os
+os.environ['S3_ENDPOINT_URL'] = 'https://s3.<your region>.backblazeb2.com'
+os.environ["AWS_ACCESS_KEY_ID"] = '<Backblaze key ID>'
+os.environ["AWS_SECRET_ACCESS_KEY"] = '<Backblaze application key>'
+```
+
+```{code-tab} sh
+export S3_ENDPOINT_URL='https://s3.<your region>.backblazeb2.com'
+export AWS_ACCESS_KEY_ID='<Backblaze key ID>'
+export AWS_SECRET_ACCESS_KEY='<Backblaze application key>'
+```
+````
+
+Note that even with S3 compatible object stores, URLs should be of the form `s3://<bucket name>/<path within the bucket>`, instead of `<endpoint url>/<bucket name>/<path within the bucket>` or in the example of Backblaze `b2://<bucket name>/<path within the bucket>`.
```

Comment:
```suggestion
Note that even with S3-compatible object stores, URLs must be of the form `s3://<bucket name>/<path within the bucket>`, instead of `<endpoint url>/<bucket name>/<path within the bucket>` or in the example of Backblaze `b2://<bucket name>/<path within the bucket>`.
```

Reply:
Sorry, I am a little confused; in the case of Backblaze, would the remote path start with `b2` or `s3`?

Reply:
They should follow `s3` paths instead of `b2`.

Reply:
Clarified this sentence.